pyvecorg/views.py

```diff
@@ -11,18 +11,15 @@
 data = load_data()
 
 
-@app.route('/')
-def index_redirect():
-    if request.accept_languages.best_match(['en', 'cs', 'sk']) == 'en':
-        return redirect(url_for('index', lang='en'))
-    return redirect(url_for('index', lang='cs'))
-
-
 @app.route('/favicon.ico')
 def favicon():
     static = os.path.join(app.root_path, 'static')
-    return send_from_directory(static, 'favicon.ico',
-                               mimetype='image/vnd.microsoft.icon')
+    return send_from_directory(static, 'favicon.ico', mimetype='image/x-icon')
```

Comment:
According to the Internet Assigned Numbers Authority (IANA), all .ico file falls under the MIME type [image/vnd.microsoft.icon](http://www.iana.org/assignments/media-types/image/vnd.microsoft.icon) which is what Travis tells you anyway.

Reply:
Oh! I understood the error message the other way around 🤦‍♂️ Thanks for the help!

Reply:
(my comment in 029dad2 being a proof 😄 )